mlos_bench/mlos_bench/tunables/tunable.py

```diff
@@ -300,7 +343,7 @@ def range(self) -> Union[Tuple[int, int], Tuple[float, float]]:
         return self._range
 
     @property
-    def categorical_values(self) -> List[str]:
+    def categorical_values(self) -> List[Optional[str]]:
```

Comment:
I am not really happy with the fact that we now have to so closely named `categorical_value` and `categorical_values` properties. Granted, mypy type checks it, but the naming is still confusing. Can't think of any better name, though. Maybe, just `categories` instead of `categorical_values`?

Reply:
I like it.

Reply:
Hmm, I wonder if we should do another PR and change `categorical_value` to `category` so that we have `categories` and `category` now?
@motus, thoughts?

Reply:
Add a PR for this in #334.  Lemme know what you think there.